### DISCUSSION TITLE
several games require full testing.

### DISCUSSION THREAD

**Discussion by Robomikel** [Robomikel/Steam-Server-Manager#222] [General]
I have tested as many games as I could. although, some bugs have slipped by me. Any testers are greatly appreciated. Testing install, logging, MCrcon, SourceMod, Backup, Gamedig, Details, etc...

**Comment by wbatzle** [comment]
I volunteer to test 7 Days to Die.

**Comment by Robomikel** [comment]
> I volunteer to test 7 Days to Die.

That would be great, thanks
